app/views/teams/index.html.slim

```diff
@@ -3,11 +3,15 @@ nav.actions
     li = link_to 'My Team', current_student.current_team, class: 'btn btn-default btn-sm' if current_student.current_team
     li = link_to icon('plus', 'New Team'), new_team_path, class: 'btn btn-primary btn-sm' if can? :create, Team.new(season: Season.current)
   div.pull-right.dropdown
-    button.dropdown-toggle.form-control  data-toggle="dropdown"
+    button.btn.btn-default.dropdown-toggle data-toggle="dropdown"
       | Past Teams
       span.caret
```

Comment:
Just tried it out locally. Works perfect, but is missing a whitespace between "past teams" and the caret 👾 

It's only a subtle difference, but I think it makes it look way cleaner with a separation:
_without whitespace:_
![screenshot-2017-12-4 rgsoc - teams app 2](https://user-images.githubusercontent.com/3491815/33555973-aaa4c396-d902-11e7-8cf6-c4898e1b6028.png)

_with whitespace:_
![screenshot-2017-12-4 rgsoc - teams app 1](https://user-images.githubusercontent.com/3491815/33555984-b75e70a0-d902-11e7-9847-7cdde82b7b43.png)


The syntax in slim therefore is a bit weird though 😉  See [the docs on it](http://www.rubydoc.info/gems/slim/frames#Trailing_and_leading_whitespace_______)

So here it would be:
```ruby
span.caret<
```

Reply:
I was trying to figure out how to do that thanks!!🙈

Reply:
Hm, maybe you forgot to push the change? I don't see it 🙈

It's literally this syntax:
```
span.caret<
```

In the changes I just see a "normal" whitespace after the `Past Teams` - this unfortunately does not work with all browsers plus most editors (mine for instance) remove trailing whitespace so this won't be around for too long 😞 

Could you update to use the slim syntax? Then we're good to merge.